index.js

```diff
@@ -143,8 +143,12 @@ function Construct(options, callback) {
           }
           fs.writeSync(out, page.slug);
         } else {
+          var pageLevel = (1.0 - page.level / 10);
+          if (pageLevel < 0) {
+            pageLevel = 0.0;
```

Comment:
I think you should bottom out at 0.1 just in case 0 is ignored. Although currently I believe Google ignores this field entirely.